src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -204,11 +208,12 @@ def run(self, args, manage_args):
                 args=' '.join(shlex_quote(arg) for arg in manage_args),
             )
         )
-        args.server = 'django_manage:0'
         if args.tmux:
+            args.server = args.server or 'django_manage:0'
```

Comment:
Oh, okay, you found a mistake in the help text then. It may have been that way at some point, but currently runs on the first django_manage machine. Can you change it to say "on the (first) django_manage machine or the server you specify" or something like that?

Reply:
Thanks @dannyroberts . Updated.